core/templates/App.ts

```diff
@@ -152,7 +152,7 @@ angular.module('oppia').config([
       'SubtopicPageContentsObjectFactory', 'SubtopicPageObjectFactory',
       'SubtopicViewerBackendApiService', 'SuggestionThreadObjectFactory',
       'SuggestionsService', 'TextInputRulesService',
-      'ThreadMessageObjectFactory', 'ThreadMessageSummaryObjectFactory',
+      'ThreadMessage', 'ThreadMessageSummary',
```

Comment:
I don't think you'll need to add it here... You can import and use the class (instead of doing require), in the respective files right?

Reply:
removed it